week-2-practise/level-3/4-slice.js

```diff
@@ -5,7 +5,8 @@
 // - insert the replacement value at the provided index
 
 function replace(arr, index, value) {
-  return; // complete this statement
+  newArr = Object.assign([], arr, {[index]: value});
```

Comment:
You need to remember to add `var` in front of variables when you declare them

Reply:
I don't think using `Object.assign` is the best way to do this. That method is usually only used for objects, not arrays.

I would look at using one or more of the array methods like `.splice()` and `.slice()`